src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/AddressDtoValidator.java

```diff
@@ -93,7 +94,8 @@ private void validateCountry(String parentAddressField, String country, List<Str
         if (countryNotBlank) {
             boolean isOnList = allowedCountries.contains(country);
             if (!isOnList) {
-                var validationMessage = String.format(ValidationMessages.COUNTRY_NOT_ON_LIST_ERROR_MESSAGE, country);
+                String sanitisedCountryName = Encode.forJava(country);
```

Comment:
We also need to truncate this field value. Here and in the validator below.

Reply:
I'm not sure that we need to truncate this. I cannot think of any country names over 50 characters long.

Reply:
The whole point of this work is if a malicious user is submitting bad data directly to our API to cause CH problems - in this case it would be log forging, filling the logs up or potentially causing some kind of service crash/error.

Because in the submited JSON body a very large country name could be submitted (not one that features in the enum) and is output, we need to do the truncation, just as we did for Confirmation Statement.

Hope that's clear but give me a call if not.